woocommerce/wc-admin
# Add support for an explicit dismiss button to snackbar, as well as an onDismiss callback

Fixes #5437

This adds support for an explicit dismiss button to the snackbar, which disables the default behaviour of clicking anywhere on the snackbar to dismiss it. It also adds a callback for when the snackbar is dismissed, which is used by the `CustomerEffortScore` component to stop displaying snackbars for the current action.

### Accessibility


### Screenshots
![image](https://user-images.githubusercontent.com/224531/99035433-60f21a00-25cb-11eb-9b85-ee48537d23e2.png)

### Detailed test instructions:

- Delete the `woocommerce_ces_shown_for_actions` option
- Add a product
- The snackbar should appear
- Clicking the snackbar body should not dismiss the snackbar
- Clicking the cross should dismiss the snackbar
- Add another product
- The snackbar should _not_ appear

### Changelog Note:

Dev: Add support for an explicit dismiss button to snackbar, as well as an onDismiss callback
